### DISCUSSION TITLE
Remap ESC to jk

### DISCUSSION THREAD

**Discussion by GitisAwesome** [neovim/neovim#24654] [Q&A] [answered]
I have used Neovim before but I do not know how to remap esc to jk. 
Please help,

Thank you.

**Comment by MSmaili** [comment, marked as answer]
vim.keymap.set(vimMode,keyMap_you_want_to_map,what_you_want_to_do,options)

For example for maping jk to ESC usually you need in insert mode.
```
vim.keymap.set( "i", "jk", "<esc>") 
```
